doc/centos-tutorial.txt

```diff
@@ -0,0 +1,141 @@
+This document outlines the necessary steps to take to install and compile MANA
+on a new CentOS machine, as well as launch, checkpoint and restart a given MPI
+application.
+
+Root privileges are required to install many of the necessary packages, and this
+tutorial will assume you have them.
+
+A. INSTALLING AND COMPILING MANA
+
+1. Installing essential packages - this section can be skipped depending on the
+existing packages on your machine.
+
+  # yum install -y git
+  # yum groupinstall 'Development Tools'
+  # yum install -y mlocate vim wget
+  # yum install -y centos-release-scl
+  # yum install -y devtoolset-8-gcc devtoolset-8-gcc-c++
+  # echo 'source scl_source enable devtoolset-8' > ~/.bashrc
+  # source ~/.bashrc
+  # yum install -y glibc-static.x86_64
+  # wget \
+      https://www.mpich.org/static/downloads/3.3.2/mpich-3.3.2.tar.gz \
+      --no-check-certificate
+  # tar xzf mpich-3.3.2.tar.gz
+  # cd mpich-3.3.2
+  # ./configure
+  # make -j install
+  # yum install -y libxml2-static zlib-static xz-devel.x86_64 yum-utils.noarch
+  # mkdir liblzma_TEMP && cd liblzma_TEMP
+  # yumdownloader --source xz-devel
+  # rpm2cpio xz-5.2.2-1.el7.src.rpm | cpio -idv
+  # tar xf xz-5.2.2.tar.gz
+  # cd xz-5.2.2
+  # patch -Np1 < ../xz-5.2.2-compat-libs.patch
+  # patch -Np1 < ../xz-5.2.2-man-page-day.patch
+  # ./configure --enable-static
+  # make
+  # cp ./src/liblzma/.libs/liblzma.a /usr/lib64/
+  # cd ~
+
+2. Installing the MANA directory and compiling MANA
+
+  $ git clone https://github.com/mpickpt/mana.git
+  $ cd mana
+  $ git checkout feature/centos
+  
+Here, we need to make some changes to the configure-mana file - these changes
+will eventually be included in a new pull request and merged into this branch.
+For now, it is included as a patch in the root directory.
+
+  $ git apply configure-mana.patch
+  $ ./configure-mana
+  $ make -j mana
+
+This should build MANA smoothly.
+
+B. LAUNCHING, CHECKPOINTING AND RESTARTING A MPI APPLICATION
+
+We'll first add the bin directory to PATH. Ensure that these commands are done
+from within the root directory of MANA.
+
+  $ echo 'export PATH=$PATH:{PATH TO MANA}/bin' >> ~/.bashrc
+  $ source ~/.bashrc
+
+By default, this should be /home/{YOUR USERNAME}/mana.
```

Comment:
Unclear.  Are you saying that by default, `{PATH TO MANA}` is `/home/{YOUR USERNAME}/mana`?  Since the user is "ensuring that these commands are done", I don't understand "By default, this should be ...".  Is it about a default in how "the user ensures that these commands are done"?  Perhaps you intended to use a phrase such as "Typically,  `{PATH TO MANA}` will be ..."./

Reply:
you are correct, this is phrased awkwardly.